product_docs/docs/migration_toolkit/55/07_invoking_mtk/08_mtk_command_options.mdx

```diff
@@ -450,21 +450,21 @@ The right side specifies a condition that must be true for each row migrated.
 
 For example, this code migrates only those countries with a `COUNTRY_ID` value that isn't equal to `AR`: 
 
-### `COUNTRIES=COUNTRY_ID<>'AR'`
+`COUNTRIES=COUNTRY_ID<>'AR'`
```

Comment:
@mlewandowski-edb, you are right. The heading formatting made me think this was a "suboption" under `-filterProp,` but I am now realizing the formatting was wrong in a few places. I turned this into normal text. 

I also changed the heading levels of `-customColTypeMapping` and `-customColTypeMappingFile` to be at the same level as the other migration options. At first, I thought these were "suboptions" too, but I now realize they are normal migration options that should be at the same level. 

Can you confirm my understanding is correct?